docs/components/generic/_index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Generic Component"
+linkTitle: "Generic"
+childTitleEndOverwrite: "Generic Component"
+weight: 55
+type: "docs"
+description: "Use a generic component to implement a component that does not fit any of the other APIs."
+tags: ["generic", "components"]
+icon: "/icons/components.png"
+images: ["/icons/components.png"]
+no_list: true
+# SMEs:
+---
+
+The _generic_ component type is for custom components that do not fit into any of the other component {{< glossary_tooltip term_id="subtype" text="subtypes" >}}.
+
+There are no built-in generic component models (other than `fake`).
+Generic is only for [modular resource](/extend/modular-resources/) models that don't fit into any of the typical categories.
+
+Use cases:
+
+- You want to implement a new model of component as a modular resource, and your device isn't an arm, a base, a board, a camera, or any of the other [built-in Viam component types](/components/).
+- You are using someone else's modular resource, which they implemented with the generic subtype.
+
+{{% alert title="Important" color="note" %}}
+
+The generic component API only supports the `DoCommand` method.
+If you use the generic subtype, your module needs to define any and all component functionality and pass it through `DoCommand`.
+
+Whenever possible, it is best to use an [existing component API](/components/) instead of generic so that you do not have to replicate code.
+If you want to use most of an existing API but need just a few other functions, try using the `DoCommand` endpoint and extra parameters to add custom functionality to an existing subtype.
+
+{{% /alert %}}
+
+## Configuration
+
+The generic component supports the following models.
```

Comment:
This may have to change in the template
```suggestion
For configuration information, click on one of the supported generic model:
```

Reply:
Yeah, also not all of them are standard still.